src/emm/services/github_service.py

```diff
@@ -0,0 +1,52 @@
+"""Service for working with github CLI."""
+from pathlib import Path
+from typing import List, Optional
+
+from plumbum import local
+
+
+class GithubService:
+    """A service for interacting with github CLI."""
+
+    def __init__(self) -> None:
+        """Initialize the service."""
+        pass
```

Comment:
If the `__init__` method isn't doing anything, we shouldn't define it. But most services that are meant to wrap a command, use the `__init__` function to wrap the command, that makes it easy to inject a mock for testing. The way this service is currently written, you will have to patch the `local.cmd.gh` object, which I prefer to avoid.